data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/LogstashMappingModel.java

```diff
@@ -55,4 +59,6 @@ public String getAttributesMapperClass() {
     public String getCustomPluginMapperClass() {
         return customPluginMapperClass;
     }
+
+    public Map<String, Object> getDefaultSettings() { return defaultSettings; }
```

Comment:
This should be formatted to sit on multiple lines. In IntelliJ, it sometimes shows these like they are on a single line, even if they are multiple.

Reply:
Yes it did, I will update the formatting.